pkg/system/phase4_configuring.go

```diff
@@ -718,6 +726,56 @@ func (r *Reconciler) prepareAWSBackingStore() error {
 	return nil
 }
 
+func (r *Reconciler) prepareAWSSTSBackingStore() error {
+	region, err := util.GetAWSRegion()
```

Comment:
all this logic seems irrelevant here. you should already have the BS spec in noobaa CR, so just create it